nautobot/docs/release-notes/version-2.0.md

```diff
@@ -206,6 +206,12 @@ Existing prefixes with a status of "Container" will be migrated to the "Containe
 
 The "Container" status will be removed and all prefixes will be migrated to the "Active" status if it exists. If the "Active" status was deleted, prefixes will be migrated to the first available prefix status in the database that is not "Container".
 
+#### Aggregate model Migrated to Prefix ([#3302](https://github.com/nautobot/nautobot/issues/3302))
```

Comment:
Nit: I'd prefer for us to keep the subsections under `### Changed` in alphabetical order just so we have some sort of consistent pattern. Looks like several of the existing subsections are already violating this pattern but let's at least not make it worse :-)